lib/xlsx_writer/sheet.rb

```diff
@@ -97,9 +97,13 @@ def local_id
     def rid
       "rId#{ndx + 1}"
     end
+
+    def filename
+      "sheet#{ndx}.xml"
+    end
     
     def relative_path
-      "xl/worksheets/sheet#{ndx}.xml"
+      "xl/worksheets/#{filename}"
```

Comment:
@seamusabshere absolute_path builds from relative_path, it seemed logical to build relative_path from filename.